pkg/crc/config/types.go

```diff
@@ -10,20 +10,22 @@ type Storage interface {
 }
 
 type Schema interface {
-	AddSetting(name string, defValue interface{}, validationFn ValidationFnType, callbackFn SetFn)
+	AddSetting(name string, defValue interface{}, validationFn ValidationFnType, callbackFn SetFn, help string)
 }
 
 type Setting struct {
 	Name         string
 	defaultValue interface{}
 	validationFn ValidationFnType
 	callbackFn   SetFn
+	Help         string
 }
 
 type SettingValue struct {
 	Value     interface{}
 	Invalid   bool
 	IsDefault bool
+	Help      string
```

Comment:
For what it's worth, this feels odd to have in `SettingValue`. Have you tried adding a new `AllSettings()` helper which would return a `[]Settings`?

Reply:
I've done this in https://github.com/cfergeau/crc/commit/ef1ef9b5e8ef97a9d3485452973d02c12929e538 and this makes `configurableFields` much nicer to read. Feel free to squash this in the relevant commits if you'd like, or I can send this as a follow-up if you prefer.